external/lein-repo/src/lein_repo/plugin.clj

```diff
@@ -223,6 +228,11 @@
                 :eval-in :subprocess
                 :jvm-opts ^:replace ["-Xmx600m" "-XX:MaxPermSize=256m"])))))
 
+(defn merge-test-profile
+  "Can't figure out how to make the task respect the :test profile, so
+  just merge the :test profile map into the project"
```

Comment:
I spent some effort getting it to work via `(leiningen.core.project/merge-profiles project [:test :leiningen/test])` like `lein test` does.